lib/kontena/actors/haproxy_config_generator.rb

```diff
@@ -66,7 +66,7 @@ def generate_service(node)
         when :virtual_hosts
           service.virtual_hosts = c.value.split(',').compact
         when :virtual_path
-          service.virtual_path = c.value unless c.value.empty?
+          service.virtual_paths = c.value.split(',').compact if c.value
```

Comment:
I also wonder if having whitespace-separated values would be better than comma-separated in general?

Using `c.value.split` might even work with something like:

```yaml
env:
  KONTENA_LB_VIRTUAL_PATH: |
   /foo
   /bar
```

(assuming that envs/labels don't break with multi-line values)

Reply:
I would keep comma-separated list for now just for consistency (see virtual hosts).

Reply:
Nothing wrong with comma-separated paths though, particularly for consistency with `KONTENA_LB_VIRTUAL_HOSTS`.

I personally just prefer the whitespace syntax over the commas, and you can't use whitespace in the URLs anyways without breaking the haproxy config as-is.